snippets/olmv1-on-cluster-catalog-queries.adoc

```diff
@@ -0,0 +1,7 @@
+// Text snippet included in the following modules:
+//
+// * olmv1-finding-operators-to-install.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+Currently, {olmv1-first} does not support querying on-cluster catalogs managed by catalogd. In OLM v1, you must use the `opm` and `jq` CLI tools to query the catalog registry.
```

Comment:
```suggestion
Currently in {olmv1-first}, you cannot query on-cluster catalogs managed by catalogd. In OLM v1, you must use the `opm` and `jq` CLI tools to query the catalog registry.
```
Just a tweak to make both sentences in this snippet address the reader ("you") instead of focusing on the product.